README.md

```diff
@@ -3,19 +3,61 @@ Clapper
 
 Synchronize setup and deployment for the Director.
 
-Configuration Validation
-========================
+Ansible-based Validations
+-------------------------
+
+Recently, we focused on writing automated validations that require
+little human setup or interference. These are driven by
+[Ansible](http://ansible.com/) we plan to provide an API that will let
+us use these from the TripleO web UI and command line clients.
+
+### Running a validation
+
+For now, you need to run them manually. All the validations live in
+the [ansible-tests/validations](ansible-tests/validations) directory.
+
+To run a validation you need to:
```

Comment:
As you wish. My point was actually more about not duplicating the documentation. The requirement.txt is useful to ensure we all end-up with the same version of ansible. This is especially important since ansible 2.0 was recently released and there is some incompatibility with older ansible playbooks. I have not yet checked our playbooks work with ansible 2.0.